src/Minsk/Minsk.csproj

```diff
@@ -13,7 +13,7 @@
     <DefaultItemExcludes>$(DefaultItemExcludes);$(GeneratedSources)</DefaultItemExcludes>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(DesignTimeBuild)' == 'true' OR '$(BuildingProject)' != 'true'">
```

Comment:
`$(BuildingProject)` is used with the old project system. The new one uses `$(DesignTimeBuild)`. We're mostly going to use it with the new project system. So, It's probably ok to omit it?

Reply:
Here's my old AZP job: https://dev.azure.com/nirindev/Minsk/_build/results?buildId=29&view=results

This is yours: https://dev.azure.com/nirindev/Minsk/_build/results?buildId=36&view=results

Both are successful? OK, Now, I'm confused?!

Reply:
![](https://media0.giphy.com/media/jPAdK8Nfzzwt2/giphy.gif)

Reply:
> Both are successful? OK, Now, I'm confused?!

I'm such an idiot 🤦‍♂️. The generated file is emitted only when in debug config and not in release config which is what the CI is on.